src/main/java/uk/gov/hmcts/reform/managecase/client/datastore/CaseUserRole.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.managecase.client.datastore;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.util.List;
+
+@Getter
+@Builder
+@AllArgsConstructor
+public class CaseUserRole {
+
+    @JsonProperty("case_id")
+    private String caseId;
+
+    @JsonProperty("user_id")
+    private String userId;
+
+    @JsonProperty("case_role")
+    private List<String> caseRole;
```

Comment:
well, this would be a just a single String. Multiple role assignments should be handled with a list of instances of this type, where some attributes can repeatedly have the same value. Please see LLD details for `Add Case-Assigned Users and Roles`. @mattnayler - also for your info as well.

Reply:
I coded as per initial call. Haven't checked 8606 LLD after that. Will make change